docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc

```diff
@@ -228,41 +222,36 @@ You can set a run rule off for a code category in either a default configuration
 
 [.task]
 
-[#review-fail-scans-and-suggestions-on-vcs]
-=== Review fail scans and suggestions on VCS (Version Control System)
+[#review-scan-result-and-suggestions-for-vcs]
+=== Review scan results and suggestions for VCS (Version Control System)
```

Comment:
@@tsmithv11 - Can I please request a review for this section? This is for the Classic UI and had some inconsistencies.